crates/bevy_render/src/texture/image_loader.rs

```diff
@@ -58,7 +58,7 @@ pub struct ImageLoaderSettings {
     pub format: ImageFormatSetting,
     pub is_srgb: bool,
     pub sampler: ImageSampler,
-    pub cpu_persistent_access: RenderAssetPersistencePolicy,
+    pub usage: RenderAssetUsages,
```

Comment:
For Image/Mesh, I feel like usage might be better renamed to asset_usage.

Reply:
I like the explicitness of that. I'll switch that over.